chia/wallet/did_wallet/did_wallet.py

```diff
@@ -470,12 +470,9 @@ async def load_parent(self, did_info: DIDInfo):
             did_wallet_puzzles.metadata_to_program(json.loads(self.did_info.metadata)),
         )
         wallet_node = self.wallet_state_manager.wallet_node
-        peer: WSChiaConnection = wallet_node.get_full_node_peer()
-        if peer is None:
-            raise ValueError("Could not find any peers to request puzzle and solution from")
-
         parent_coin: Coin = did_info.origin_coin
         while True:
+            peer = wallet_node.get_full_node_peer()
```

Comment:
is there a good reason to move this into the loop? it means we'll drop our reference to the peer, and then re-acquire it the next iteration. And we'll have more opportunities for this to throw

Reply:
The idea was to make sure we have a connected peer before each all since there are async calls in the loop.